test/tools/fuzzer_common.cpp

```diff
@@ -72,7 +72,16 @@ void FuzzerUtil::testCompilerJsonInterface(string const& _input, bool _optimize,
 	runCompiler(jsonCompactPrint(config), _quiet);
 }
 
-void FuzzerUtil::testCompiler(StringMap const& _input, bool _optimize, unsigned _rand)
+void FuzzerUtil::forceSMT(StringMap& _input)
+{
+	// Add SMT checker pragma if not already present in source
+	static const char* smtPragma = "pragma experimental SMTChecker;";
```

Comment:
```suggestion
	static const char* smtPragma = "pragma experimental SMTChecker;\n";
```
? not sure if readability of the generated code is important.

Reply:
Oops actually this is suboptimal and better not to do for the following reason:

Sometimes the fuzzer generates `pragma experimental SMTChecker;function f() {}` which is fine (since it enables the smt checker) but if we append a `newline` character, the match is not found and another smt checker pragma is added. Of course if we could do `sourceUnit.second += smtPragma + '\n'` instead